core/XH.ts

```diff
@@ -92,8 +93,8 @@ export class XHApi {
     exceptionHandler: ExceptionHandler = new ExceptionHandler();
 
     //----------------------------------------------------------------------------------------------
-    // Metadata - set via webpack.DefinePlugin at build time.
-    // See @xh/hoist-dev-utils/configureWebpack.
+    // Metadata - the `xhFooBar` values on the right hand of these assignments are injected at build
```

Comment:
prefer `xhXxx`

Reply:
Will swap to that - it's what I typed first and then seemed like a lot of X's, but agree :)